firstvoices/backend/search/indices/dictionary_entry_document.py

```diff
@@ -166,3 +167,66 @@ def update_notes(sender, instance, **kwargs):
                 dictionary_entry.id,
             )
         )
+
+
+@receiver(post_save, sender=Site)
+def update_site(sender, instance, **kwargs):
```

Comment:
Would there be any benefit to using the site id in the index instead of the site slug? Id is immutable

Reply:
It would be nice to at least verify that the slug was changed before attempting to update all the indexed items for the site. The site is more likely to be updated just to change the banner image or something